recipes/BEHST/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: behst
+  version: 0.4
+
+source:
+  fn: a2a25d973909.zip
+  url: https://bitbucket.org/hoffmanlab/behst/get/a2a25d973909.zip
+  sha256: cf4cc7179e6311f72d350c4fd408930b367b568f4ca77469474c4e5044ffd015
+
+build:
+  number: 1
```

Comment:
https://conda.io/docs/building/meta-yaml.html#build-number-and-string: "The build number should be incremented for new builds of the same version. The number defaults to zero."

If you re-build version 0.4 with some fixes to the recipe, then you would increase the build number.